### ISSUE TITLE
Datepicker Aria Label Warnings

### ISSUE THREAD

**Issue by kmgilbert100** [angular/material#5028]
I get the following warning whenever I use datepickers

```ARIA: Attribute " aria-label ", required for accessibility, is missing on node: ```

Adding the aria label attr to the **<md-datepicker>** doesnt seem to do it either.

**Comment by marcysutton** [comment]
Can you provide more information, including a Codepen or Plunker demo? There are no warnings on the demos. Also, which version of Angular Material?

**Comment by JCummings01** [comment]
I'm having a similar issue - when I put my email <input> in a <md-input-container>, I get this error. I added 'aria-label="..."' to make the error disappear but the input box does not function.

**Comment by marcysutton** [comment]
Hi, we need to see an example to help you. Can you provide a Codepen demo that shows your issue?

**Comment by berikin** [comment]
http://codepen.io/anon/pen/RWBzYw

This codepen has two md-datepickers with aria-label, with Google Chrome  46.0.2490.80 for Mac the two md-datepickers outputs this warning in the console:

ARIA: Attribute " aria-label ", required for accessibility, is missing on node: 

    <button class="md-datepicker-button md-icon-button md-button md-ink-ripple" ng-transclude=""     type="button" tabindex="-1" aria-hidden="true" ng-click="ctrl.openCalendarPane($event)"><md-icon class="md-datepicker-calendar-icon ng-scope ng-isolate-scope" md-svg-icon="md-calendar" aria-hidden="true"><svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%" viewBox="0 0 24 24" fit="" preserveAspectRatio="xMidYMid meet" style="pointer-events: none; display: block;"><path d="M19 3h-1V1h-2v2H8V1H6v2H5c-1.11 0-1.99.9-1.99 2L3 19c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm0 16H5V8h14v11zM7 10h5v5H7z"></path></svg></md-icon></button>

**Comment by marcysutton** [comment]
Ahh, I see the issue–the utility isn't respecting `aria-hidden:true` and `tabindex:"-1"` on that button. There are other keyboard/screen reader affordances, so the button is okay to be hidden...the utility needs to be updated to ignore buttons that aren't in the accessibility tree (like that one).

**Comment by jelbourn** [comment]
@marcysutton thanks for checking that out. I'll try to get around to fixing that (unless you want to take it, of course).
